lib/matplotlib/table.py

```diff
@@ -146,6 +147,57 @@ def set_text_props(self, **kwargs):
         self._text.update(kwargs)
 
 
+class CustomCell(Cell):
+    """
+    A subclass of Cell where the sides may be visibly toggled.
+
+    """
+
+    EDGES = 'BRTL'
+    EDGE_ALIASES = {'open':         '',
```

Comment:
Should they be prepended with an underscore like "_edges"? To signify that they shouldn't be accessed or should they be accessible?